server/src/testFixtures/java/io/crate/testing/SQLExecutor.java

```diff
@@ -485,7 +487,8 @@ public Builder addPartitionedTable(String createTableStmt, Settings customSettin
                     this.columnOidSupplier != null ? this.columnOidSupplier : mdBuilder.columnOidSupplier();
             Map<String, Object> mapping = TestingHelpers.toMapping(columnOidSupplier, boundCreateTable);
 
-            XContentBuilder mappingBuilder = JsonXContent.builder().map(mapping);
+            XContentBuilder mappingBuilder =
```

Comment:
Otherwise this assertion: https://github.com/crate/crate/blob/master/server/src/main/java/io/crate/metadata/cluster/DDLClusterStateHelpers.java#L142 fails, as there is no `"default"` entry, and the map is null.